clone.html

```diff
@@ -0,0 +1,233 @@
+          </div>
+          <div class="carousel-item">
+            <img class="d-block w-100 bwhite" src="./starter-code/img/slider-2.jpg" alt="Second Slide">
+            <div class="carousel-caption d-none d-md-block">
+              <h3>Etiam mauris tortor, pharetra quis lobortis in, pharetra in diam</h3>
+              <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc.</p>
+              <button class="btn btn-outline-light" type="button">Read More</button>
+            </div>
+          </div>
+          <div class="carousel-item">
+            <img class="d-block w-100 bwhite" src="./starter-code/img/slider-3.jpg" alt="Third Slide">
+            <div class="carousel-caption d-none d-md-block">
+              <h3>Etiam mauris tortor, pharetra quis lobortis in, pharetra in diam</h3>
+              <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc.</p>
+              <button class="btn btn-outline-light" type="button">Read More</button>
+            </div>
+          </div>
+          <div class="carousel-item">
+            <img class="d-block w-100 bwhite" src="./starter-code/img/slider-4.jpg" alt="Fourth Slide">
+            <div class="carousel-caption d-none d-md-block">
+              <h3>Etiam mauris tortor, pharetra quis lobortis in, pharetra in diam</h3>
+              <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc. </p>
+              <button class="btn btn-outline-light" type="button">Read More</button>
+            </div>
+          </div>
+          
+        </div>
+        <a href="#slider4" class="carousel-control-prev" data-slide="prev">
+          <span class="carousel-control-prev-icon"></span>
+        </a>
+
+        <a href="#slider4" class="carousel-control-next" data-slide="next">
+          <span class="carousel-control-next-icon"></span>
+        </a>
+      </div>
+    </div>
+    <div class="colored">
+      <div class="container">
+        <div class="row">
+          <div class="col">
+            <p class="icons">🖱</p>
```

Comment:
The icons are not exactly the same as in the original, but that is just a minor detail.